rex/utilities/utilities.py

```diff
@@ -590,3 +590,144 @@ def roll_timeseries(arr, timezones):
         local_arr[:, mask] = np.roll(arr[:, mask], int(tz * time_step), axis=0)
 
     return local_arr
+
+
+def get_chunk_slices(ds_dim, chunk_size):
+    """
+    Create list of chunk slices [(s_i, e_i), ...]
+
+    Parameters
+    ----------
+    ds_len : int
+        Length of dataset axis to chunk
+    chunk_size : int
+        Size of chunks
+
+    Returns
+    -------
+    chunks : list
+        List of chunk start and end positions
+        [(s_i, e_i), (s_i+1, e_i+1), ...]
```

Comment:
the title of this method would imply that this outputs a list of slices.

Reply:
that might be outdated, i'll update